### PR TITLE
feat: update builtins list

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/camunda/camunda-modeler/issues/3884

image

### CODE REVIEW

File: scripts/compileBuiltins.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ glob(MARKDOWN_SRC, function(err, files) {
 
       return rawDescriptors.flatMap(string => {
         const name = string.split('\n')[0];
-        const description = marked.parse(string.split('\n').slice(1).join('\n'));
+        let description = marked.parse(string.split('\n').slice(1).join('\n'));
+
+        description = description.replace('<MarkerCamundaExtension></MarkerCamundaExtension>', '<b>Camunda Extension</b>');
```
Comment (barmac) [inline scripts/compileBuiltins.js:22]: I'd rather expect this to be a heading `<h[n]>`.

Comment (marstamm) [reply scripts/compileBuiltins.js:22]: The "Camunda Extension" marker is can be placed within a paragraph (`p`) in the docs, making headings semantically incorrect

Comment (barmac) [reply scripts/compileBuiltins.js:22]: So this one is a badge, correct? Not a heading then.

Comment (marstamm) [reply scripts/compileBuiltins.js:22]: Yes, it looks like this in the docs:

image

Comment (barmac) [reply scripts/compileBuiltins.js:22]: Perhaps `strong` or `em` conveys the message of "this is important". The latter is even boldface per default.

Comment (marstamm) [reply scripts/compileBuiltins.js:22]: `em` seems to be semantically what we want, so let's use that one

**Review by barmac — APPROVED** [review]

**Pull request merged**